recipe/meta.yaml

```diff
@@ -17,15 +17,15 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python >=3.9
   run:
     - astropy >=5,<6
     - filelock >=3,<4
     - frictionless >=4.29,<5
     - pandas >=1,<2
     - plotly >=5,<6
     - pybtex >=0.24,<0.25
-    - python >=3.6
+    - python >=3.9
```

Comment:
@saraedum is it right that python should be included in host and in run?

Reply:
Yes, it should be in both.